### ISSUE TITLE
Release 4.1.2 - typo in package.json results in missing dependency

### ISSUE THREAD

**Issue by GooSP** [Orange-OpenSource/Orange-Boosted-Angular#115]
Hi, I'm using ng-boosted for a project which needs the version 4.2.1 (for angular version 7) but there is a typo in the `package.json` of the library:

https://github.com/Orange-OpenSource/Orange-Boosted-Angular/blob/bae49257e9bdf27d6e1528c2f0c6116f98962895/projects/ng-boosted/package.json#L17

Can you patch it to avoid the need to put dependencies manually (`swiper`) in the project.

Thanks.

**Comment by Lausselloic** [comment]
Hello,

Yes the issue have been fixed by @robertIsaac thanksto his PR #106 

We will release the version as soon as possible.

**Comment by robertIsaac** [comment]
Duplicate of #105

**Comment by BehnooshShiva** [comment]
version 1.5.5 is available. 
This version contains the correction of this bug.

**Comment by robertIsaac** [comment]
> version 5.1.1 is available.
> This version contains the correction of this bug.

then why don't we close the issue?

**Comment by GooSP** [comment]
Yea, but 5.1.1 is for angular 8 which requires a whole upgrade into the project.

**Comment by Lausselloic** [comment]
You're right, will see if there's an opportunity to release a 4.2.2 version to fix the dispo

**Comment by BehnooshShiva** [comment]
V4.2.2 is available.
@GooSP  let me know if there is any problem

**Comment by BehnooshShiva** [comment]
@GooSP 
Sorry for the problem. you can find the correction in version 4.2.3

**Comment by GooSP** [comment]
Working well now with 4.2.3, thanks :)
